### ISSUE TITLE
Annotatrix should not strip udep column if it doesn't conform to ud standards

### ISSUE THREAD

**Issue by ftyers** [jonorthwash/ud-annotatrix#341]
The old version of Annotatrix did the Right Thing.

![captura de 2019-02-23 04-16-32](https://user-images.githubusercontent.com/449545/53281526-ed86e580-3721-11e9-8211-87a9781c430b.png)

That is, the rels are there but they are marked in red.

**Comment by jonorthwash** [comment]
I assume this was an intentional "feature" in notatrix?

Perhaps there should be a "strict mode" for optionally turning on the current strict behaviour—but otherwise, yeah, we mostly don't want it to be strict like this.

**Comment by maryszmary** [comment]
For some reason, we either don't understand the issue or can't reproduce it. What happens in the current version?

**Comment by jonorthwash** [comment]
@ftyers, yeah, this bug report is confusing.  You say it strips the "udep column" (DEPREL?), but you show that it just highlights it in red.

I get the same behaviour you demonstrate and describe ("marked in red"), but don't understand why it's a problem / how it's worse than what happened before.  Could you clarify / explain what "did the Right Thing" means (and how it's different from what's happening now)?
